store/mockstore/unistore/tikv/mvcc_test.go

```diff
@@ -1800,3 +1801,68 @@ func TestAssertion(t *testing.T) {
 		kvrpcpb.Assertion_NotExist, kvrpcpb.AssertionLevel_Strict, store)
 	require.Nil(t, err)
 }
+
+// mockScanProcessor process the key/value pair.
+type mockScanProcessor struct{}
+
+// Process deals with the value.
+func (m *mockScanProcessor) Process(key, value []byte) error {
+	return nil
+}
+
+// SkipValue returns if we can skip the value.
+func (m *mockScanProcessor) SkipValue() bool {
+	return false
+}
+
+func TestRcReadCheckTS(t *testing.T) {
+	store, clean := NewTestStore("TestRcReadCheckTS", "TestRcReadCheckTS", t)
+	defer clean()
+
+	// Prepare.
+	k1 := []byte("tk1")
+	v1 := []byte("v1")
+	MustPrewriteOptimistic(k1, k1, v1, 1, 100, 0, store)
+	MustCommit(k1, 1, 2, store)
+
+	k2 := []byte("tk2")
+	v2 := []byte("v2")
+	MustPrewriteOptimistic(k2, k2, v2, 5, 100, 0, store)
+	MustCommit(k2, 5, 6, store)
+
+	k3 := []byte("tk3")
+	v3 := []byte("v3")
+	MustPrewriteOptimistic(k3, k3, v3, 1, 100, 0, store)
+
+	// Test point get with RcReadCheckTS.
+	reader := store.newReqCtx().getDBReader()
```

Comment:
Reading by `DBReader` means we directly read from badger, what about reading from `store` so that the locks will be checked.

Reply:
It's changed to use `mvccStore`, PTAL.